eppo_core/src/event_ingestion/event_ingestion.rs

```diff
@@ -7,30 +7,32 @@ use uuid::Uuid;
 use crate::{background::BackgroundRuntime, sdk_key::SdkKey};
 
 use super::{
-    auto_flusher, batcher, delivery, event_delivery::EventDelivery, queued_event::QueuedEvent,
-    retry, BatchedMessage, Event,
+    auto_flusher, batcher,
+    delivery::{self, DeliveryConfig},
+    event_delivery::EventDelivery,
+    BatchedMessage, Event,
 };
 
 #[derive(Debug, Clone)]
 #[non_exhaustive]
 pub struct EventIngestionConfig {
     pub sdk_key: SdkKey,
     pub ingestion_url: Url,
+    pub max_queue_size: usize,
     pub delivery_interval: Duration,
-    pub retry_interval: Duration,
-    pub max_retry_delay: Duration,
-    pub max_retries: u32,
     pub batch_size: usize,
-    pub max_queue_size: usize,
+    pub max_retries: u32,
+    pub base_retry_delay: Duration,
```

Comment:
The naming of `retry_interval` and `max_retry_delay` didn't quite connect. I renamed `retry_interval` to `base_retry_delay`, so I hope `base_retry_delay` + `max_retry_delay` is easier to understand